src/zenml/models/pipeline_build_models.py

```diff
@@ -40,12 +40,20 @@ class BuildItem(BaseModel):
 
     Attributes:
         image: The image name or digest.
+        dockerfile: The dockerfile used to build the image.
+        requirements: The pip requirements installed in the image.
```

Comment:
Actually not valid contents of a single requirements file as it might contain multiple indexes etc. This is just multiple requirements files appended, but I can adjust the docstring a little